### ISSUE TITLE
Choosing appropriate hyperparameters for the loss function

### ISSUE THREAD

**Issue by inderjeetnair** [v-mipeng/LexiconNER#13]
Hello,
Thanks for sharing this code. This code is extremely easy to use and very readable. I wanted to know if there are some practical considerations for deciding upon the hyperparameters associated to the loss function. Particularly, what are your recommendations for choosing appropriate value for m(class balance rate), beta, prior and gamma for a given dataset?
